dashboard/app/controllers/api/v1/pd/workshop_enrollments_controller.rb

```diff
@@ -5,4 +5,16 @@ class Api::V1::Pd::WorkshopEnrollmentsController < ::ApplicationController
   def index
     render json: @workshop.enrollments, each_serializer: Api::V1::Pd::WorkshopEnrollmentSerializer
   end
+
+  # DELETE /api/v1/pd/workshops/1/enrollments/1
+  def destroy
+    enrollment = @workshop.enrollments.find_by(id: params[:id])
+    unless enrollment
+      head :not_found
```

Comment:
I'll need a migration to add deleted_at so I'd like to do it in a separate PR.

I also have a work item to make workshops soft-delete, so I can do them both together.
I was thinking about upgrading our `paranoia` gem and leveraging that. I know you had some issues with it in `Followers` related to unique constraints but these tables don't have that and the newer version might be fixed (our paranoia gem is quite old).